src/index.js

```diff
@@ -19,12 +21,27 @@ exports.fetchManifest = async () => {
                             reject(e);
                         }
                     } else {
-                        reject(new Error(`Status: ${res.statusCode}\n${json}`));
+                        reject(new Error(`Url: ${url}\n${json}`));
                     }
                 })
                 .on('error', (err) => reject(err));
         }).on('error', (err) => reject(err));
     });
 };
 
+exports.fetchManifest = async () => {
+    const manifest = await fetchResource(MANIFEST_URL);
+
+    try {
+        const consoleReadmeSuffix = await fetchResource(CONSOLE_README_SUFFIX_URL);
+        const localReadmeSuffix = await fetchResource(LOCAL_README_SUFFIX_URL);
+        manifest.consoleReadmeSuffix = consoleReadmeSuffix;
+        manifest.localReadmeSuffix = localReadmeSuffix;
+    } catch (e) {
+        // Could not fetch the suffixes
```

Comment:
I'd probably just log it here in case we'd need to debug it.

```suggestion
        // Could not fetch the suffixes
        console.error('Could not fetch the suffixes.', e);
```